pages/common/fastboot.md

```diff
@@ -0,0 +1,27 @@
+# fastboot
+
+> Communicate to connected Android devices when in bootloader mode (the one place adb doesn't work).
+
+- Unlock bootloader:
+
+`fastboot oem unlock`
+
+- Re-lock the bootloader:
+
+`fastboot oem lock`
+
+- Reboot device from fastboot mode to fastboot mode again:
+
+`fastboot reboot bootloader`
+
+- Flash ASDF.zip:
```

Comment:
What precisely is `ASDF.zip` in this case?

Reply:
It is an image which is to be flashed. I think "Flash a given image" and then change ASDF.zip in the example to file.zip would be better.